frontend/package.json

```diff
@@ -23,7 +23,7 @@
         "build:esbuild": "DEBUG=0 node ./build.mjs",
         "build:tailwind": "pnpm --filter=@posthog/tailwind build",
         "watch:tailwind": "pnpm --filter=@posthog/tailwind start",
-        "start": "concurrently -n ESBUILD,TYPEGEN,TAILWIND -c yellow,green,blue \"pnpm start-http\" \"pnpm run typegen:watch\" \"sleep 40 && pnpm watch:tailwind\"",
+        "start": "concurrently -n ESBUILD,TYPEGEN,TAILWIND -c yellow,green,blue \"pnpm start-http\" \"pnpm run typegen:watch\" \"pnpm watch:tailwind\"",
```

Comment:
i had to change this for my build to work

specifically changed here https://github.com/PostHog/posthog/pull/29685